.github/dependabot.yml

```diff
@@ -5,10 +5,14 @@ updates:
     directory: /
     schedule:
       interval: weekly
+    cooldown:
+      default-days: 7
 
   - package-ecosystem: github-actions
     directory: /
     schedule:
       interval: weekly
     open-pull-requests-limit: 99
     rebase-strategy: "disabled"
+    cooldown:
+      default-days: 7
```

Comment:
https://www.fixyourlineendings.com/ ? 😀

Reply:
Hah, zizmor's auto fix didn't do this. I guess I need to look into making it add line endings.